src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1577,14 +1609,13 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	}
 
 	private doLayout(dimensions: IEditorTitleControlDimensions, options?: IMultiEditorTabsControlLayoutOptions): void {
-
-		// Only layout if we have valid tab index and dimensions
-		const activeTabAndIndex = this.group.activeEditor ? this.getTabAndIndex(this.group.activeEditor) : undefined;
-		if (activeTabAndIndex && dimensions.container !== Dimension.None && dimensions.available !== Dimension.None) {
-
+		// Only layout when there is an active editor
+		// The active editor might be located in a different tab row
+		if (this.groupViewer.activeEditor && dimensions.container !== Dimension.None && dimensions.available !== Dimension.None) {
```

Comment:
I believe we do not need the `this.groupViewer.activeEditor` check here. Because if there is no editor, this method will not be called given `isVisible === false`

Reply:
If this is an assumption we can make, you are right